requirements.txt

```diff
@@ -12,6 +12,7 @@ requests >= 2.25.0
 
```

Comment:
For epython, the requirements.txt is _only_ used for development and test. The real dependencies for installing epython are found in the setup.cfg per pypi requirements.